src/main/java/uk/gov/di/ipv/service/ConfigurationService.java

```diff
@@ -21,11 +21,16 @@ public String getAuthCodesTableName() {
         return System.getenv("AUTH_CODES_TABLE_NAME");
     }
 
+    public String getUserIssuedCredentialTableName() {
+        return System.getenv("USER_ISSUED_CREDENTIALS_TABLE_NAME");
+    }
+
+
     public String getAccessTokensTableName() { return System.getenv("ACCESS_TOKENS_TABLE_NAME"); }
 
     public long getBearerAccessTokenTtl() {
         return System.getenv().containsKey("BEARER_TOKEN_TTL")
-            ?  Long.parseLong(System.getenv("BEARER_TOKEN_TTL"))
-            : DEFAULT_BEARER_TOKEN_TTL_IN_SECS;
+                ?  Long.parseLong(System.getenv("BEARER_TOKEN_TTL"))
+                : DEFAULT_BEARER_TOKEN_TTL_IN_SECS;
```

Comment:
Instead of using `containKey`, then `getenv`, you could try:
````
return Optional.of(System.getenv("BEARER_TOKEN_TTL"))
                .map(Long::valueOf)
                .orElse(DEFAULT_BEARER_TOKEN_TTL_IN_SECS);
````
They're both 3 lines long though 😄